src/ht.jl

```diff
@@ -0,0 +1,17 @@
+# TODO: add docstrings
+function HorwitzThompson_HartleyRaoApproximation(y::AbstractVector, design::AbstractSurveyDesign, HT_total)
```

Comment:
Why are these modifications part of this PR?

Reply:
These come from my git mess up. I merged this branch with the `design_update` branch at some point in a wrong way.